src/nrtk_explorer/library/coco_utils.py

```diff
@@ -59,7 +59,7 @@ def convert_from_ground_truth_to_second_arg(dataset_annotations, dataset):
 def convert_from_predictions_to_first_arg(predictions, dataset, ids):
     """Convert predictions to COCOScorer format"""
     predictions = convert_from_predictions_to_second_arg(predictions)
-    categories = {cat["name"]: cat["id"] for cat in dataset["categories"]}
+    categories = {cat["name"]: cat["id"] for cat in dataset.cats.values()}
```

Comment:
Here I'm confident this can be done more efficiently with the vectorized interface (especially in the SQL context):

```python
id_to_name = dataset.categories().lookup('name', keepid=True)
categories = {name: id for id, name in id_to_name.items()}
```

This won't be much different in the json dataset backend, but in the SQL context it will be able to ignore all other fields a category might have and just look at the name and id column.